### DISCUSSION TITLE
Questions about GNN pre training

### DISCUSSION THREAD

**Discussion by HelloWorldLTY** [pyg-team/pytorch_geometric#5470] [General]
Hi, I wonder if PYG provides any pre-training strategies for GNN pre-training? Thanks a lot.

**Comment by rusty1s** [comment]
We currently only provide examples for unsupervised learning, but don't have any examples of pre-training methods recently proposed. I would be very happy to include such an example. Alternatively, please take a look [here](https://github.com/snap-stanford/pretrain-gnns).
